CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.2.0
+
+## Features and improvements
+* Add *-sort-key layout property for circle, fill, line ([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
+* Add map-specific access tokens ([#8364](https://github.com/mapbox/mapbox-gl-js/pull/8364))
+* Make requirements for text offset props more precise ([#8418](https://github.com/mapbox/mapbox-gl-js/pull/8418))
+* Accommodate prefers-reduced-motion settings in browser ([#8494](https://github.com/mapbox/mapbox-gl-js/pull/8494))
+* Add Map visualizePitch option that tilts the compass as the map pitches ([#8208](https://github.com/mapbox/mapbox-gl-js/issues/8208), fixed by [#8296](https://github.com/mapbox/mapbox-gl-js/pull/8296)) (h/t [pakastin](https://github.com/pakastin))
+* Explicit source options now take precedence over TileJSON ([#8232](https://github.com/mapbox/mapbox-gl-js/pull/8232)) (h/t [jingsam](https://github.com/jingsam))
+* Expose convertFilter API in the style specification ([#8493](https://github.com/mapbox/mapbox-gl-js/pull/8493)
+
+## Bug fixes
+* Fix rendering of opaque pass layers over heatmap and fill-extrusion layers ([#8440](https://github.com/mapbox/mapbox-gl-js/pull/8440))
+* Favor previous anchor only when still in the `text-variable-anchor` options ([#8473](https://github.com/mapbox/mapbox-gl-js/pull/8473))
```

Comment:
I don't think this PR name is clear as a changelog entry and would prefer:

> Fix bug where previous anchors set in `text-variable-anchor` were still preferred after anchor options were reset

Reply:
the change emphasizes that previous positions would get prioritized _only_ if they're also present in the updated style value, but i'll figure out a way to articulate that